test/suite/helper_tiered.py

```diff
@@ -145,13 +149,33 @@ def gen_tiered_storage_sources(random_prefix='', test_name='', tiered_only=False
             bucket_prefix2 = generate_s3_prefix(random_prefix, test_name),
             num_ops=20,
             ss_name = 's3_store')),
+        ('azure_store', dict(is_tiered = True,
```

Comment:
Remove this please, since we aren't using it.

Reply:
addressed https://github.com/wiredtiger/wiredtiger/pull/8707/commits/61b13a704e7d58933ead288cc550b6b4492ad90c